src/util/triplea/mapXmlCreator/MapXmlData.java

```diff
@@ -45,7 +45,7 @@ public MapXmlData() {
     initialize();
   }
 
-  @SuppressWarnings("unused")
+  
   // Maybe for testing needed
```

Comment:
This @SuppressWarnings Announciation was triggering a "unnececary SuppressWarnings" warning...